### PR TITLE
Add rescript support

### PR DESCRIPTION
**Pull request by amiralies**
this PR adds support for the [rescript](http://rescript-lang.org) language

image

### CODE REVIEW

File: lib/rouge/lexers/rescript.rb [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    load_lexer 'ocaml/common.rb'
+
+    class ReScript < OCamlCommon
+      title "ReScript"
+      desc "The ReScript programming language (rescript-lang.org)"
+      tag 'rescript'
+      filenames '*.res', '*.resi'
+      mimetypes 'text/x-rescript'
+
+      operators = %r([;_!$%&*+/<=>@^|-]+)
```
Comment (pyrmont) [inline lib/rouge/lexers/rescript.rb:15]: Is there any reason to make this a class-level instance variable? It's only used in one place, right?

**Review by pyrmont — CHANGES_REQUESTED** [review]
Just one question:

File: lib/rouge/lexers/rescript.rb (hunk 1 above)
Comment (amiralies) [reply lib/rouge/lexers/rescript.rb:15]: You're right

**Review by pyrmont — APPROVED** [review]

**Comment by pyrmont** [comment]
@amiralies Thanks for adding this. This will be part of the next release of Rouge. That's scheduled to go out on Tuesday (or Wednesday depending on your time zone) of this week 🎉

**Pull request merged**